lib/buildkite/test_collector/uploader.rb

```diff
@@ -40,7 +44,7 @@ def self.upload(data)
       Thread.new do
         response = begin
           upload_attempts ||= 0
-          http.post_json(data)
+          @response = http.post_json(data)
```

Comment:
We can add the suite_url to the response from the `post_json` which comes from `Analytics::API::UploadsController.create` within BK.

I'm not 100% on this part, and how I am assigning the response here.. keen to get feedback on it

Reply:
Draft PR for the BK part of this here: https://github.com/buildkite/buildkite/pull/14288